spec/index.bs

```diff
@@ -2008,9 +2008,15 @@ conditions.
 <script type=idl>
 dictionary GPUShaderModuleDescriptor : GPUObjectDescriptorBase {
     required DOMString code;
+    object sourceMap;
 };
 </script>
 
+{{GPUShaderModuleDescriptor/sourceMap}}, if defined, is interpreted as a source-map-v3 format.
```

Comment:
What happens if there's a sourcemap v4?  Should there be another field that specifies the version of the sourcemap?  Or would you reuquire a new GPUShaerModuleDescriptor when that time comes?

Reply:
[from meeting] Source map spec has a version field at the top level.

Reply:
I will s/is interpreted as/MAY be interpreted as/